docs/Face-services-and-plugins.md

```diff
@@ -12,31 +12,96 @@ CompreFace supports these face services and plugins:
 ## Services
 
 To use face service you need to create it in an application on UI. The type of service depends on your application needs. Each service has its own REST API context and there is no possibility to change the service type after creation. Here is a short description of each of them:
-* Face recognition service is used for face identification. This means that you first need to upload known faces to faces collection and then recognize unknown faces among them. When you upload an unknown face, the service returns the most similar faces to it. Also, face recognition service supports verify endpoint to check if this person from face collection is the correct one. The possible cases include:
+
+### Face detection
+
+Face detection service is used to detect all faces in the image. It doesn’t recognize faces, just finds them on the image.
+
+**Cases of use**
+
+The most useful cases include face plugins for face analysis:
+  * gather statistics on how your store popular among different genders
+  * gather statistics on among what ages your event is popular
+  * get landmark information to know where customers look at
+  * gather statistics on how many customers in the store
+
+**How to use**
+
+1. On the ComperFace application page, at the bottom of the frame, click Create button.
+2. In the Create Service dialog, from the Type drop-down menu, select DETECTION.
+3. Enter the name of the service you are going to create.
+4. From the list of the services in the Services frame, select the service you created; you can use search field to filter the services.
+5. Click Test button near in the row of the service you want to launch.
+6. On the service page, open or drag-and-drop the picture to analyze.
+
+**Output**
+
+The service will display the original picture with marks near every face.
+Below the picture, you can see the Request processed, and the Response to the request.
+
+### Face recognition
+
+Face recognition service is used for face identification. This means that you first need to upload known faces to faces collection and then recognize unknown faces among them. When you upload an unknown face, the service returns the most similar faces to it. Also, face recognition service supports verify endpoint to check if this person from face collection is the correct one. 
```

Comment:
"then recognize unknown faces among them" 
Not sure, but for me, it sounds like we want to find unknown faces among known.
What if we rephrase it to:
"then match unknown faces with known uploaded faces"